script/build.js

```diff
@@ -101,7 +117,17 @@ if (options.buildtype === 'production') {
   ignoreList.push('profile/*');
   ignoreList.push('auth/*');
   ignoreList.push('education/apply-for-education-benefits-new');
+} else if (options.buildtype === 'staging') {
```

Comment:
@jkassemi: This seems like it won't really address the much larger issue of staging and prod being based off of `master` and `production`, respectively. If we're going to base them off of different branches, I don't think we need to worry about trying overly hard to keep the environments so tightly synced, as we should really consider moving to [blue-green](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) deployments or something similar (and better regression testing than the app devs noticing something wrong would help a lot too).